Autopilot/AttitudeManager/UART2.h

```diff
@@ -1,35 +1,18 @@
-/*
- * File:   UART2.h
- * Author: Mitch
- *
- * Created on June 15, 2013, 2:52 PM
+/**
+ * @file UART2.h
+ * @author Mitch Hatfield
+ * @date Jun 2013
+ * @brief Settings for UART2 communication used for telemetry link 
+ * @copyright Waterloo Aerial Robotics Group 2016 \n
+ *   https://raw.githubusercontent.com/UWARG/PICpilot/master/LICENCE 
  */
 
-#ifndef UART2_H
-#define	UART2_H
+#pragma once
+
 #include "main.h"
 
+/** Initializes the UART2 connection **/
 void InitUART2();
-void UART2_SendString(char *s);
-void UART2_SendChar(char data);
```

Comment:
This will break it. The send string and send char are still used in other parts of the code.